zh_tw.lang

```diff
@@ -0,0 +1,209 @@
+gui.xaero_confirm =確認
+gui.xaero_cancel =取消
+gui.xaero_back =返回
+gui.xaero_delete =刪除
+gui.xaero_OK =確定
+gui.xaero_add_edit =添加/編輯
+gui.xaero_new_waypoint =新的路徑點
+gui.xaero_initial =初始化
```

Comment:
Isn't "初始" more correct?

Reply:
Due to the lack of full context, it can't be determined to use "初始" or "初始化" or even "首字母", but it seems to be a noun? If so, then "首字母" is the correct one.